README.md

```diff
@@ -11,7 +11,7 @@ Albert is a unified and efficient access to your machine. Technically it is a ke
 
 ## Getting started
 
-Check the section [*Installing Albert*](https://albertlauncher.github.io/docs/installing/) in the docs for setup instructions. When you are set up take a look at the section [*Using Albert*](https://albertlauncher.github.io/docs/using/). If you have problems check the [*Troubleshooting*](https://albertlauncher.github.io/docs/faq/) section. If it does not cover your problems seek for [*help*](https://albertlauncher.github.io/help/) in one of the chats. Developers may want to check the [*Extending Albert*](https://albertlauncher.github.io/docs/extending/) section.
```

Comment:
Can you also update the other links? they all need update to remove '/docs/' in the url

Reply:
@qzhuyan, Done.  All the links work, now.  The only ones I couldn't fix were the ones in src/app/settingswidget/settingswidget.ui